app/views/authorised_to_view/authorised_to_view_search.scala.html

```diff
@@ -59,11 +60,11 @@
         @p("cf.search.authorities.p")
 
         <ul class="govuk-list govuk-list--bullet">
-            @if(isXiEoriEnabled) {
-                <li>@messages("cf.search.authorities.eori.xi")</li>
-            }else{
-                <li>@messages("cf.search.authorities.eori")</li>
-            }
+            <li>@messages(
```

Comment:
Here is one way of using pattern matching

```suggestion
@{
  val eoriMessageKey = (isEUEoriEnabled, isXiEoriEnabled) match {
    case (true, _)    => "cf.search.authorities.eori.eu"
    case (false, true) => "cf.search.authorities.eori.xi"
    case _             => "cf.search.authorities.eori"
  }
}

<ul class="govuk-list govuk-list--bullet">
  <li>@messages(eoriMessageKey)</li>
  <li>@messages("cf.search.authorities.account")</li>
</ul>
```

Reply:
ahh you need to  inline the logic and the HTML in the same `@{ ... }` block

So like this:

```
@{
    val eoriMessageKey = (isEUEoriEnabled, isXiEoriEnabled) match {
        case (true, _)     => "cf.search.authorities.eori.eu"
        case (false, true) => "cf.search.authorities.eori.xi"
        case _             => "cf.search.authorities.eori"
    }

    <ul class="govuk-list govuk-list--bullet">
        <li>@messages(eoriMessageKey)</li>
        <li>@messages("cf.search.authorities.account")</li>
    </ul>
}
```

Reply:
Legend, thank you! was doing circles for ages yesterday lol